tests/eval/mod.rs

```diff
@@ -213,7 +213,7 @@ fn print_print() -> LangResult<'static, ()> {
         machine_builder().with_env(Env::new(&mut output)).build(),
     )?;
     let output = String::from_utf8_lossy(&output);
-    assert_eq!(output, "(print 10)\n");
+    assert_eq!(output, "10\n0\n");
```

Comment:
Is that the output that we want...? I think we shouldn't be able to print things that evaluate to `Unit`. Can we catch that in the type checker?

Reply:
We could but printing `Unit` shouldn't be a problem at all. I think the real problem is that we're printing the integer representation of literals instead of some higher level representation